app/models/merchant.rb

```diff
@@ -12,4 +13,13 @@ def top_five_customers
       .order("count_transactions DESC")
       .limit(5)
    end
+
+
+   def change_status
```

Comment:
Do you still need this method?

Reply:
nope!
I had it before adding some changes the controller update method
It’s gone now!